src/components/Dashboard/Dashboard.jsx

```diff
@@ -181,6 +183,10 @@ class Dashboard extends Component {
     const { selectedWidgetIndex } = this.state;
     this.updateWidget(selectedWidgetIndex, { attribute });
   }
+  handleCommandChange(command) {
```

Comment:
There should be a blank line between class members. This is an esLint rule.

Reply:
fixed